vector/src/main/java/im/vector/app/features/userdirectory/UserListViewModel.kt

```diff
@@ -85,6 +85,11 @@ class UserListViewModel @AssistedInject constructor(
     }
 
     init {
+        // Tchap: Force user consent to true, we don't want to display the consent banner
+        if (!session.identityService().getUserConsent()) {
+            session.identityService().setUserConsent(true)
+        }
+
```

Comment:
@Florian14 can you please confirm we need to force the consent here too?

Reply:
This one is necessary but I am not sure the other one is necessary as we don't display the contact book in Tchap

Reply:
The ContactBook is not visible in tchap. We don't need to set the userConsent to true.